docs-src/docs/kognic-io/calibrations/cameras-custom.md

```diff
@@ -102,6 +102,10 @@ kognicutil wasm validate calibration.wasm
 
 ## Compilation
 
+:::caution
```

Comment:
Would this be better placed under the Rust example instead?

Reply:
The only heading that I see with rust is example with rust where I think it might get lost. If I would look for quick info on why script is failing I would look here I think. I think that this banner can be removed in a while since we have the version below so this is mostly temporary